user_guide/content/modules/creating_a_new_space.adoc

```diff
@@ -15,24 +15,24 @@ To start a new project, users must create a new Collaboration Space. Each Space
 
 . Add a unique name for the new space in the *Name* field.
 
-. Select a *Template* type from the drop-down menu. The four options are:
-.. *Agile* - Your space is centered around Agile-based plannning.
-.. *Scrum* - Your space includes an iterative and incremental Agile-based development framework. This option is similar to the **Agile** option but with more specific planning and development tracking.
-.. *Issue Tracking* - Your space is based on an issue tracking development methodology. This option is ideal for teams interested in primarily tracking issues and Tasks and resolving them.
+. Select a *Template* type from the drop-down menu. The available options are:
+//.. *Agile* - Your space is centered around Agile-based planning.
```

Comment:
Yeah, I thought it made sense to leave it this way because the other options are most likely coming back soon and we can just switch it over then because that will *probably* happen before we publish the docs. WDYT?